apps/cowswap-frontend/src/modules/hooksStore/containers/HookDappContainer/index.tsx

```diff
@@ -15,6 +15,7 @@ import { useOrderParams } from '../../hooks/useOrderParams'
 import { HookDapp, HookDappContext as HookDappContextType } from '../../types/hooks'
 import { isHookDappIframe } from '../../utils'
 import { IframeDappContainer } from '../IframeDappContainer'
+import { useHookBalancesDiff } from 'modules/hooksStore/hooks/useBalancesDiff'
```

Comment:
Nitpick: The import should be relative, since the hooks comes from the same module